cms/static/js/spec/views/settings/main_spec.js

```diff
@@ -186,5 +186,41 @@ define([
             AjaxHelpers.expectJsonRequest(requests, 'POST', urlRoot, modelData);
         });
 
+        describe("enrollmentEndEditable set to true", function() {
```

Comment:
Nit: could make this a single "describe" with a shared beforeEach (containing lines 191-194), then instantiate this.view in the individual test cases. It would remove some of the redundancy in these two describes.